assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -69,9 +149,18 @@ export const DumbManageModules = ({
         </p>
         <div className="two-columns-admin">
           <div className="column-left">
-            <SelectModulesForm lang={locale} moduleTypes={moduleTypes} modulesById={modulesById} toggleModule={toggleModule} />
+            <SelectModulesForm
+              lang={locale}
+              moduleTypes={updatedModuleTypes}
+              modulesById={modulesById}
+              toggleModule={toggleModule}
+            />
             <div className="margin-xl">
-              <Button className="button-submit button-dark" onClick={displayConfirmationModal}>
+              <Button
```

Comment:
It took me time to realize/remember that this button is here only to add text and multimedia modules. I guess it would be clearer if we had some kind of `AddTextAndMultimediaButton` component with all this logic (modal creation, etc)
We will need to rethink/refactor this component if we need there is another module that we need to add multiple times in the landing page but IMHO this whole UI needs to be totally redesigned.

Reply:
>  but IMHO this whole UI needs to be totally redesigned.

💯